packages/dsv/package.json

```diff
@@ -25,22 +25,34 @@
     "prepare": "pnpm run build",
     "prepublishOnly": "pnpm run lint",
     "pretest": "pnpm run build",
-    "test": "ava"
+    "test": "ava",
+    "test:ts": "tsc ./test/types.ts ./types/index.d.ts --noEmit"
   },
   "files": [
     "dist",
+    "types",
     "README.md",
     "LICENSE"
   ],
+  "peerDependencies": {
+    "@types/d3-dsv": "~1.2.0"
```

Comment:
This should just be a hard `dependencies` instead of peer.

Reply:
Hm. Types were installed as peer dependencies in the [Babel Plugin](https://github.com/rollup/plugins/blob/master/packages/babel/package.json#L49).

Reply:
babel/core is a peer dependency in that plugin but d3-dsv is a built in dependency here. However, we should probably update that plugin too anyways.

Reply:
:+1: